ui/src/app/machines/views/MachineList/MachineListHeader/ActionFormWrapper/DeployForm/DeployFormFields/DeployFormFields.tsx

```diff
@@ -102,6 +103,30 @@ export const DeployFormFields = (): JSX.Element => {
             ]}
             inline
           />
+          <FormikField
+            label="Cloud-init user data..."
```

Comment:
same here `user-data`. Should the "..." be `&hellip`?

Reply:
Done